src/services/block.service.ts

```diff
@@ -23,6 +24,30 @@ export class BlockService {
     return await this.blockStorage.getMinedBlockByHash(hash);
   }
 
+  public async getBlockStats(): Promise<any> {
+    const allBlocks = await this.blockStorage.getAllBlocks();
+    let averageBlockTime = 0;
+    let txThroughput = 0;
+    if (allBlocks && allBlocks.length > 1) {
+      allBlocks.sort((a, b) => b.timestamp - a.timestamp);
+      const timeDiff = (allBlocks[0].timestamp - allBlocks[allBlocks.length - 1].timestamp) / 1000;
```

Comment:
I think `allBlocks[allBlocks.length - 1].timestamp > allBlocks[0].timestamp`
So should reserve those two to avoid negative value

Reply:
See the line above this. i'm doing b - a calculation. So it shall be ordered from big to small in the list